src/frontend/admin/src/components/presentational/translatable-content/TranslatableContent.tsx

```diff
@@ -41,11 +52,15 @@ export function TranslatableContent({ ...props }: PropsWithChildren<Props>) {
       in the current language and not in the current language forced. by the TranslatableContent component
      */
     window.onbeforeunload = () => {
+      const oldDjangoLanguage = getSavedDjangoLanguage() + "";
```

Comment:
If the key return null, the function already return an empty string.
```suggestion
      const oldDjangoLanguage = getSavedDjangoLanguage();
```